contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/plugins/formulas/MarkdownFormula.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+
+  <span :class="{editing: editing}">
+    <span ref="mathField">
+      <slot></slot>
+    </span>
+  </span>
+
+</template>
+
+<script>
+
+  import Vue from 'vue';
+  import vueCustomElement from 'vue-custom-element';
+  import '../../mathquill/mathquill.js';
+  import css from '!css-loader!less-loader!./style.less';
```

Comment:
`vue-custom-element` doesn't use inline Vue styles, so it's necessary to actually load the processed styles directly from webpack...  I'll make another comment showing how they are used.